docs/file-formats/xml-data-file.md

```diff
@@ -997,22 +997,20 @@ They can have the following predefined properties:
 | `relatesTo`        | 0-n         | resptr-prop   |             |
 
 The segment bounds must be entered in seconds. 
-Decimal values for fractions of a second are allowed, e.g. `1.4:2`.
+Decimal values for fractions of a second are allowed, e.g. `1.4`.
 The segment from 0h 10min 00s to 0h 20min 00s would thus be represented by the interval `600:1200`.
 
 Example:
 
 ```xml
-<video-segment label="Segment of video_thing_1" id="video_segment_0" permissions="res-default">
-    <resptr-prop name="isSegmentOf">
-        <resptr permissions="prop-default">video_thing_1</resptr>
-    </resptr-prop>
-    <interval-prop name="hasSegmentBounds">
-        <interval permissions="prop-default">10:20</interval>
-    </interval-prop>
-    <text-prop name="hasComment">
-        <text encoding="xml" permissions="prop-default">Optional comment</text>
-    </text-prop>
+<video-segment label="Video Segment" id="video_segment_1">
```

Comment:
Is it also possible to have comments in the elements?

Reply:
done with https://github.com/dasch-swiss/dsp-tools/commit/2f0c162cf8ad2892edb1a3195630d7e48004079a